django_netjsonconfig/base/admin.py

```diff
@@ -90,6 +91,9 @@ def get_urls(self):
             url(r'^preview/$',
                 self.admin_site.admin_view(self.preview_view),
                 name='{0}_preview'.format(url_prefix)),
+            url(r'^context/(?P<pk>[^/]+)/$',
```

Comment:
what is the final URL generated by this line?
We must ensure it's something like: `/device/<pk>/context/`

Reply:
it is: `admin/django_netjsonconfig/device/context/<pk>/`

Reply:
ok yes, let's do that, as we do with the `schema.json` view